snactor/loader/group.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+from snactor.loader import create_actor, load
+from snactor.registry import get_registered_actors, must_get_actor
+from snactor.utils import get_chan
+
+
+class UnresolvedDependenciesError(Exception):
+    """
+    UnresolvedDependenciesError is thrown when the dependencies between actors can't be automatically resolved
+    """
+    pass
+
+
+def _order_by_channels(channels, actors):
+    result = []
+    while actors:
+        scheduled = []
+        for idx, actor in enumerate(actors):
+            for channel in actor[0].inputs:
+                if channels[channel['name']]['producers']:
+                    break
+            else:
+                # for channel in actor[0].inputs:
+                #    channels[channel['name']]['consumers'].remove(actor[0].name)
```

Comment:
Could we remove this?

Reply:
Probably yes, I wasn't sure about the use case here